src/components/shared/editable-text-input-list/editable-text-input-list.tsx

```diff
@@ -3,37 +3,65 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Button, Form, InputGroup } from "react-bootstrap";
+import { Button, Form, InputGroup } from 'react-bootstrap';
+import PageBlock from '../page-block/page-block';
 
 interface EditableTextInputListProps {
+  title: string;
   items: any[];
+  readOnly?: boolean;
+  ordered?: boolean;
   add: (val: any) => any;
   remove: (idx: number) => any;
   update: (idx: number, val: any) => any;
 }
 
 const EditableTextInputList: React.FC<EditableTextInputListProps> = ({
+  title,
   items,
+  readOnly,
+  ordered,
   add,
   remove,
   update
 }) => {
+  let listPrepared = items.map((item, index: number) =>
+    !readOnly ? (
+      <div key={index} className={'mb-2'}>
+        <InputGroup>
+          <Form.Control
+            type="text"
+            value={item.toString()}
+            onChange={(e) => {
+              update(index, e.target.value);
+            }}
+          />
+          <Button type="button" variant="danger" onClick={() => remove(index)}>
+            X
+          </Button>
+        </InputGroup>
+      </div>
+    ) : (
+      <li key={index}>{item}</li>
+    )
+  );
```

Comment:
`Form.Control` has a prop to make it readonly, so there's no need for the ternary operator. Also, if we're making the list readOnly, we'd probably want a ternary operator on the remove button instead (if `readOnly`, don't render anything aka `<></>`; otherwise, render the remove button)

Reply:
True, but the readOnly style looks like a greyed-out textbox. Do we want this kind of styling? From a UI/UX standpoint I feel like it would be a little confusing.

Reply:
Not really sure tbh, I'd ask the PMs about that in the product-request or product channel. Although, this is just MVP so I've been operating under the impression that UI doesn't need to be perfect and so I've let things like this just slide.

Reply:
Sent a message in #product-request.

Reply:
personally I prefer the <li> and it doesn't really add any complexity i think